test/TestUtilities/Test.Utility/TestPackagesCore.cs

```diff
@@ -174,16 +174,18 @@ public static TempFile GetLegacyFolderPackage()
             return file;
         }
 
-        public static TempFile GetLegacyTestPackage()
+        public static TempFile GetLegacyTestPackage(string packageName="test")
         {
             var file = new TempFile();
 
             using (var zip = new ZipArchive(File.Create(file), ZipArchiveMode.Create))
             {
-                zip.AddEntry("lib/test.dll", ZeroContent);
-                zip.AddEntry("lib/net40/test40.dll", ZeroContent);
-                zip.AddEntry("lib/net40/test40b.dll", ZeroContent);
-                zip.AddEntry("lib/net45/test45.dll", ZeroContent);
+                zip.AddEntry("lib/"+ packageName + ".dll", ZeroContent);
+                zip.AddEntry("lib/net40/"+ packageName + "40.dll", ZeroContent);
+                zip.AddEntry("lib/net40/"+ packageName + "40b.dll", ZeroContent);
+                zip.AddEntry("lib/net45/"+ packageName + "45.dll", ZeroContent);
+                zip.AddEntry("lib/net40/en/" + packageName + ".resources.dll", ZeroContent);
+                zip.AddEntry("lib/net45/en-GB/" + packageName + ".resources.dll", ZeroContent);
```

Comment:
I'm not a fan of modifying a generic test helper method to test one specific feature, as it will affect all tests, including those not related to the feature. I think tests related to satellite assemblies should be more explicit about needing those files, so people unfamiliar with the feature can more easily understand what the test is testing.

Reply:
Added new method to generate satellite assemblies while this case is a bit specific.